A66-otel-stats.md

```diff
@@ -0,0 +1,566 @@
+
+Buckets for histograms in default views should be as follows -
+
+*   Latency : 0, 0.00001, 0.00005, 0.0001, 0.0003, 0.0006, 0.0008, 0.001, 0.002,
+    0.003, 0.004, 0.005, 0.006, 0.008, 0.01, 0.013, 0.016, 0.02, 0.025, 0.03,
+    0.04, 0.05, 0.065, 0.08, 0.1, 0.13, 0.16, 0.2, 0.25, 0.3, 0.4, 0.5, 0.65,
+    0.8, 1, 2, 5, 10, 20, 50, 100
+*   Size : 0, 1024, 2048, 4096, 16384, 65536, 262144, 1048576, 4194304,
+    16777216, 67108864, 268435456, 1073741824, 4294967296
+*   Count : 0, 1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192,
+    16384, 32768, 65536
+
+These buckets were chosen to maintain compatibility with the gRPC OpenCensus
+spec. The OpenTelemetry API has added an experimental feature for
+[advice](https://opentelemetry.io/docs/specs/otel/metrics/api/#instrument-advice)
+that would allow the gRPC library to provide these buckets as a hint. Since this
+is still an experimental feature and not yet implemented in all languages, it is
+up to the user of the gRPC OpenTelemetry plugin to choose the right bucket
+boundaries and set it through the
+[OpenTelemetry SDK](https://opentelemetry.io/docs/specs/otel/metrics/sdk/#view).
+
+Note that, according to an
+[OpenTelemetry proposal on stability](https://docs.google.com/document/d/1Nvcf1wio7nDUVcrXxVUN_f8MNmcs0OzVAZLvlth1lYY/edit#heading=h.dy1cg9doaq26),
+changes to bucket boundaries may not be considered as breaking. Depending on the
+proposal, this recommendation would change to use `ExponentialHistogram`s
+instead, which would allow for automatic adjustments of the scale to better fit
+the data.
+
+#### Attributes
+
+*   `grpc.method` : Full gRPC method name, including package, service and
+    method, e.g. "google.bigtable.v2.Bigtable/CheckAndMutateRow". Note that gRPC
+    servers can receive arbitrary method names, i.e., method names that have not
+    been registered in advance with the server. This normally results in those
+    RPCs being rejected with an UNIMPLEMENTED status. Some gRPC implementations
+    allow servers to handle such generic method names. Since the stats plugin
+    would be recording all of these RPCs, this could open up the server to
+    malicious attacks that result in metrics being stored with a high
+    cardinality. To prevent this, unregistered/generic method names should by
+    default be reported with "other" value instead. Implementations should
+    provide the option to override this behavior to allow recording generic
+    method names as well.
+*   `grpc.status` : gRPC server status code received, e.g. "OK", "CANCELLED",
+    "DEADLINE_EXCEEDED".
+    [(Full list)](https://grpc.github.io/grpc/core/md_doc_statuscodes.html)
+*   `grpc.target` : Canonicalized target URI used when creating gRPC Channel,
+    e.g. "dns:///pubsub.googleapis.com:443", "xds:///helloworld-gke:8000".
+    Canonicalized target URI is the form with the scheme included if the user
+    didn't mention the scheme ([scheme]:///[target]). For channels such as
```

Comment:
I think this should be `scheme://[authority]/path`?  No brackets on scheme or path as those are required.

Reply:
Done